scripts/build-image.sh

```diff
@@ -32,9 +34,9 @@ shift 5
 registries=("${@}")
 
 if [ "${with_root_context}" = "false" ] ; then
-  image_tag="$("${root_dir}/images/scripts/make-image-tag.sh" "${image_dir}")"
```

Comment:
This has slipped in form #37, as I was testing this script with slightly different directory structure.